packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1,6 +1,12 @@
 {
   "403": "Forbidden",
   "500": "Internal Server Error",
+  "5_mins":"5 minutes",
+  "15_mins":"15 minutes",
+  "1_hour":"One hour",
+  "6_hours":"Six hours",
+  "12_hours":"Twelve hours",
+  "24_hours":"Twenty-four hours",
```

Comment:
Instead of 24 hours, wouldn't "1 day" sound better? Wdyt?

Reply:
Changed to 1 day!